### ISSUE TITLE
Don't swallow Exceptions that come from ViewModels 

### ISSUE THREAD

**Issue by sdoward** [alter-ego/androidbound#160]
As far as I understand this lib catches all exceptions that come from the ViewModels and prints out a line. This is dangerous as it can hide issues in the consumers code. It stops crash reports being sent and doesn't give good feedback to developers using this lib.

**Comment by sdoward** [comment]
An alternative to this would be to accept a simple interface that can be used as a crash/exception reporter

```
interface ExceptionReporter {

    logException(Throwable throwable);

}
```

This way consumers of the lib can get a good understanding of issues that might occur with their code base

**Comment by mrsasha** [comment]
fixed in 0.9.1
